test/Interop/Cxx/operators/member-inline-irgen.swift

```diff
@@ -1,4 +1,7 @@
 // RUN: %target-swift-emit-ir %s -I %S/Inputs -enable-cxx-interop | %FileCheck %s
+//
+// We can't yet call member functions correctly on Windows.
+// XFAIL: OS=windows-msvc
```

Comment:
I disabled these two tests for now, because the fact that we can't call member functions correctly on Windows is somewhat unrelated.

See SR-13129.

Reply:
Could you add a link to the bug in the comment?

Reply:
SGTM.

Reply:
Oops, I missed this. Done in https://github.com/apple/swift/pull/32699.